From 81487ad9de2726b916f17f06274add5120318fcc Mon Sep 17 00:00:00 2001
From: John Greene <jogreene@redhat.com>
Date: Thu, 14 Aug 2014 14:42:12 -0400
Subject: [ethernet] e1000e: Enclose e1000e_pm_thaw() with CONFIG_PM_SLEEP

Message-id: <1408027348-9610-19-git-send-email-jogreene@redhat.com>
Patchwork-id: 87708
O-Subject: [RHEL7.1 PATCH 18/34] e1000e: Enclose e1000e_pm_thaw() with CONFIG_PM_SLEEP
Bugzilla: 1091119
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>

Fix following compilation warning:
drivers/net/ethernet/intel/e1000e/netdev.c:6238:12: warning
'e1000e_pm_thaw' defined but not used [-Wunused-function]
 static int e1000e_pm_thaw(struct device *dev)
            ^
Signed-off-by: Hiroaki SHIMODA <shimoda.hiroaki@gmail.com>
Tested-by: Aaron Brown <aaron.f.brown@intel.com>
Signed-off-by: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
(cherry picked from commit 3e7986f67c0e444b2419f25c48815e17ebbab836)

BZ: 1091119
Testing: Acceptable, see BZ
Brew: https://brewweb.devel.redhat.com/taskinfo?taskID=7834849
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/intel/e1000e/netdev.c b/drivers/net/ethernet/intel/e1000e/netdev.c
index 2a27dfe..13e1baf 100644
--- a/drivers/net/ethernet/intel/e1000e/netdev.c
+++ b/drivers/net/ethernet/intel/e1000e/netdev.c
@@ -6251,6 +6251,7 @@ static int __e1000_resume(struct pci_dev *pdev)
  return 0;
 }
 
+#ifdef CONFIG_PM_SLEEP
 static int e1000e_pm_thaw(struct device *dev)
 {
  struct net_device *netdev = pci_get_drvdata(to_pci_dev(dev));
@@ -6271,7 +6272,6 @@ static int e1000e_pm_thaw(struct device *dev)
  return 0;
 }
 
-#ifdef CONFIG_PM_SLEEP
 static int e1000e_pm_suspend(struct device *dev)
 {
  struct pci_dev *pdev = to_pci_dev(dev);
-- 
1.7.1